### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by camilog**
General description of RaTA DNS, with the new modules (Fievel, Gopher and Remy).
More information will be added in wiki page in future days.

### CODE REVIEW

**Comment by eacha** [comment]
Mention the wiki for more information

File: README.md [hunk 1]
```diff
@@ -1,130 +1,37 @@
...
-```json
-{
-  "type": "QueriesPerSecond",
-  "serverId": "server1",
-  "data": {
-    "qps": 426.08993504359
-  }
-}
-```
+Fievel is the first step in the pipeline RaTA DNS. Captures the data (DNS packets) coming from the network interface, a .pcap file or standard input. After the capture, the data goes through various preprocessing stages called Preliminary Reducers (PreR), that has as goal reduce the quantity of information to send to the next module.
```
Comment (fmontoto) [inline README.md]: pipeline of RaTA DNS/ RaTA DNS' pipeline

Comment (fmontoto) [inline README.md]: s/quantity/amount

**Comment by Zeeerooo** [comment]
I'm not sure about the english, but LGTM.

**Comment by storress** [comment]
You could put the module links (fievel, gopher, remy) in a list

**Comment by camilog** [comment]
@storress done

**Comment by storress** [comment]
LGTM

**Comment by camilog** [comment]
@fmontoto changed a bit the explanation, mentioning fievel, gopher and remy as components

**Comment by fmontoto** [comment]
LGTM

**Pull request merged**